practice-two/src/index.html

```diff
@@ -164,7 +164,22 @@ <h2 class="heading-section text-color-base testimonials__header-text">What Clien
 
     <!-- START PROJECTS SECTION -->
     <section></section>
-    <!-- END TPROJECTS SECTION-->
+    <!-- END PROJECTS SECTION-->
+
+    <!-- START CTA SECTION -->
+    <section class="cta text-color-secondary">
+      <div class="container cta__wrapper">
+        <div class="cta__header">
+          <h2 class="heading-section">Start Selling Instantly</h2>
+          <p class="text-normal cta__text">Start Selling With Your Pre-paid account. No deposit needed.</p>
+        </div>
+        <form action="javascript:void(0)" class="cta__form">
+            <input type="email" name="email" placeholder="Email" class="form-input cta__input">
+            <button type="submit" class="btn-secondary btn cta__button">Register</button>
```

Comment:
The order of class should be `btn btn-secondary cta__button`

You should use a modifier class instead element class `cta__button`